components/webpack/webpack.config.js

```diff
@@ -72,19 +89,30 @@ module.exports = async function (env, argv) {
 
   return {
     entry,
-    devtool: isDevMode ? '#inline-source-map' : false,
+    devtool: isDevMode ? 'inline-source-map' : false,
     output,
     resolve,
     optimization: {
       // Define NO_CONCATENATE for analyzing module size.
       concatenateModules: !process.env.NO_CONCATENATE
     },
-    plugins: process.env.DEPFILE_SOURCE_NAME ? [
-      new GenerateDepfilePlugin({
+    experiments: {
+      syncWebAssembly: true
+    },
+    plugins: [
+      process.env.DEPFILE_SOURCE_NAME && new GenerateDepfilePlugin({
         depfilePath: process.env.DEPFILE_PATH,
         depfileSourceName: process.env.DEPFILE_SOURCE_NAME
-      })
-    ] : [],
+      }),
+      // Provide globals from NodeJs polyfills
+      new webpack.ProvidePlugin({
+        'Buffer': ['buffer', 'Buffer'],
+        'process': 'process/browser.js'
+      }),
+      prefixReplacer('chrome://resources/brave/fonts', path.join(process.env.ROOT_GEN_DIR, 'brave/ui/webui/resources/fonts')),
```

Comment:
Webpack5 only supports a couple of URL prefixes in imports. Unfortunately we still need the `resolve.alias` entry for these, it's just no longer enough on it's own.

Reply:
@fallaciousreasoning instead of repeating them, and having to define them in 2 places, should we generate these from `pathMap`? Something like
```ts
Object.keys(pathMap).filter(p => p.startsWith('chrome://')).map(p => prefixReplacer(p, pathMap[p]))
```